codersdk/workspaces.go

```diff
@@ -37,6 +37,8 @@ type CreateWorkspaceBuildRequest struct {
 	Transition        WorkspaceTransition `json:"transition" validate:"oneof=create start stop delete,required"`
 	DryRun            bool                `json:"dry_run,omitempty"`
 	ProvisionerState  []byte              `json:"state,omitempty"`
+	// ParameterValues are optional. It will write params to the 'workspace' scope.
```

Comment:
You should explain the behavior that it doesn't delete the old values, but any present values will overwrite any existing values with the same name.

Reply:
That is true, old values remain. I hope that is not a problem :thinking:

Reply:
This is true... I wonder if I should delete the unused params :thinking:. I can add this in another PR and test it more